roles/cockpit/meta/main.yml

```diff
@@ -12,7 +12,4 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- role: os_firewall
-  os_firewall_allow:
-  - service: cockpit-ws
-    port: 9090/tcp
+- role: lib_os_firewall
```

Comment:
Let's set this work for a refactor.  The README is where the documentation for this module lives.  I'll leave it here until we can merge the README doc into the module doc.